spec/rubocop/cop/lint/missing_super_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::MissingSuper do
```

Comment:
I think this is missing some test cases. I don't see any tests that cover `method_missing`